packages/terra-framework-docs/src/terra-dev-site/doc/popup/example/PopupWithArrow.jsx

```diff
@@ -18,25 +19,66 @@ function PopupArrow() {
 
   const handleRequestClose = () => {
     setOpen(false);
+    buttonElement.current.focus(); // Return focus to the button when the popup closes
   };
 
+  const handleKeyDown = (event) => {
+    if (event.key === 'ArrowDown') {
+      event.preventDefault();
+      setFocusedIndex((prevIndex) => (prevIndex + 1) % 2);
+    } else if (event.key === 'ArrowUp') {
+      event.preventDefault();
+      setFocusedIndex((prevIndex) => (prevIndex - 1 + 2) % 2);
+    }
+  };
+
+  useEffect(() => {
+    if (open) {
+      setFocusedIndex(0);
+    }
+  }, [open]);
+
+
+  useEffect(() => {
+    if (open && focusedIndex !== -1) {
+      const listItems = document.querySelectorAll('.terra-popup-content a');
+      console.log("CHEKING ",listItems)
```

Comment:
```suggestion
      console.log("CHEKING ",listItems)
```
console.logs should be removed